q2cli/builtin/tools.py

```diff
@@ -1456,14 +1458,15 @@ def supplement_replay(
 @click.option(
     '--annotation-type',
     required=True,
-    type=click.Choice(['Note'], case_sensitive=True),
+    type=click.Choice(ANNOTATION_TYPE_DICT, case_sensitive=True),
```

Comment:
```suggestion
    type=click.Choice(list(ANNOTATION_TYPE_DICT.keys()), case_sensitive=True),
```

Does using just the entire dict here work as expected? I feel like there's likely to be some amount of unexpected behavior from this. I think what we want here is explicitly just a list of the keys from the dict, and we should probably explicitly get that.